src/parse/ris_live.go

```diff
@@ -93,23 +94,28 @@ func ParseRisLiveData(msgChannel chan common.BGPMessage) error {
 	fmt.Println("made connection")
 
 	/* Subscribe */
-	subscription1 := RisMessage{"ris_subscribe", &RisMessageData{"", "0.0.0.0/0"}}
-	//TODO: connect subscription to config file
-	//		keep an array of subscriptions
+	subscriptionPrefixes := strings.Split(configPrefixes, ",")
+
+	if len(subscriptionPrefixes) == 0 {
+		subscriptionPrefixes = append(subscriptionPrefixes, "0.0.0.0/0")
+	}
+
+	for _, prefix := range subscriptionPrefixes {
+		subscription := RisMessage{"ris_subscribe", &RisMessageData{"", prefix}}
+		out, err := json.Marshal(subscription)
+		if err != nil {
+			return errors.New("Error marshalling subscription message, " + err.Error())
+		}
+		log.Println("Subscribing to: ", prefix)
+		conn.WriteMessage(websocket.TextMessage, out)
+	}
 
 	// alternatives:
```

Comment:
these are examples left behind from quite a while ago; you can safely remove these